### ISSUE TITLE
compose error and docker-compose is outdated?

### ISSUE THREAD

**Issue by LotekHeavy** [mschnitzer/ark-survival-ascended-linux-container-image#96]
I have a few ASA servers configured, and when I, for example, try to add additional mods and modify the docker-compose.yml file (e.g., extending the "ASA_START_PARAMS"), and then run docker-compose up -d, I encounter a Python error: "KeyError: 'ContainerConfig'". (I  use Ubuntu 24.04 LTS, Python 3.12.x and Docker version 27.4.1 and docker-compose version 1.29.2)

When I switch to the newer syntax by running docker compose up -d, I see the following error:
Error response from daemon: Conflict. The container name "/f6d2084dfe90_asa-server-2" is already in use by container "f6d2084dfe90fcbb89152c186128c7daab6c4c74261de2ab3ca0709aa8530a62". You have to remove (or rename) that container to be able to reuse that name.

The solution I found was to run docker compose down first and then re-run the docker compose up -d command. Is this the correct approach? (Sorry, I’m a docker newbie.)

By the way, it seems like docker-compose is outdated (according to [this blog post](https://www.docker.com/blog/announcing-compose-v2-general-availability/)), so using docker compose is now the preferred method. Is that correct?

**Comment by mschnitzer** [comment]
You are most likely fine if you replace all `docker-compose` commands with `docker compose` on your distribution.

**Comment by LotekHeavy** [comment]
yes a expirmented in the last view days a bit and seems to work, thx
